2018-10-25 20:30:39 [ ERROR ] [ AbstractClient$1.java :167(run)]  [DUBBO] client reconnect to 192.168.1.100:20881 find error . url: dubbo://192.168.1.100:20881/com.dape.upms.rpc.api.UpmsApiService?anyhost=true&application=dape-upms-server&check=false&codec=dubbo&default.check=false&dubbo=2.5.6&generic=false&heartbeat=60000&interface=com.dape.upms.rpc.api.UpmsApiService&methods=selectUpmsOrganizationByExample,insertUpmsLogSelective,selectUpmsRoleByUpmsUserId,selectUpmsRolePermisstionByUpmsRoleId,selectUpmsSystemByExample,selectUpmsPermissionByUpmsUserIdByCache,selectUpmsUserByUsername,selectUpmsRoleByUpmsUserIdByCache,selectUpmsPermissionByUpmsUserId,selectUpmsUserPermissionByUpmsUserId&mock=true&pid=11444&remote.timestamp=1540453075612&revision=1.0&side=consumer&timeout=10000&timestamp=1540454336901, dubbo version: 2.5.6, current host: 192.168.1.100
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.1.100:20881/com.dape.upms.rpc.api.UpmsApiService?anyhost=true&application=dape-upms-server&check=false&codec=dubbo&default.check=false&dubbo=2.5.6&generic=false&heartbeat=60000&interface=com.dape.upms.rpc.api.UpmsApiService&methods=selectUpmsOrganizationByExample,insertUpmsLogSelective,selectUpmsRoleByUpmsUserId,selectUpmsRolePermisstionByUpmsRoleId,selectUpmsSystemByExample,selectUpmsPermissionByUpmsUserIdByCache,selectUpmsUserByUsername,selectUpmsRoleByUpmsUserIdByCache,selectUpmsPermissionByUpmsUserId,selectUpmsUserPermissionByUpmsUserId&mock=true&pid=11444&remote.timestamp=1540453075612&revision=1.0&side=consumer&timeout=10000&timestamp=1540454336901) failed to connect to server /192.168.1.100:20881, error message is:No route to host: no further information
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:124)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:268)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:157)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.net.NoRouteToHostException: No route to host: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at org.jboss.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.connect(NioClientSocketPipelineSink.java:384)
	at org.jboss.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processSelectedKeys(NioClientSocketPipelineSink.java:354)
	at org.jboss.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:276)
	... 3 more
2018-10-25 23:37:55 [ ERROR ] [ AbstractClient$1.java :167(run)]  [DUBBO] client reconnect to 192.168.1.100:20881 find error . url: dubbo://192.168.1.100:20881/com.dape.upms.rpc.api.UpmsApiService?anyhost=true&application=dape-upms-server&check=false&codec=dubbo&default.check=false&dubbo=2.5.6&generic=false&heartbeat=60000&interface=com.dape.upms.rpc.api.UpmsApiService&methods=selectUpmsOrganizationByExample,insertUpmsLogSelective,selectUpmsRoleByUpmsUserId,selectUpmsRolePermisstionByUpmsRoleId,selectUpmsSystemByExample,selectUpmsPermissionByUpmsUserIdByCache,selectUpmsUserByUsername,selectUpmsRoleByUpmsUserIdByCache,selectUpmsPermissionByUpmsUserId,selectUpmsUserPermissionByUpmsUserId&mock=true&pid=11444&remote.timestamp=1540453075612&revision=1.0&side=consumer&timeout=10000&timestamp=1540480971509, dubbo version: 2.5.6, current host: 192.168.1.101
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.1.100:20881/com.dape.upms.rpc.api.UpmsApiService?anyhost=true&application=dape-upms-server&check=false&codec=dubbo&default.check=false&dubbo=2.5.6&generic=false&heartbeat=60000&interface=com.dape.upms.rpc.api.UpmsApiService&methods=selectUpmsOrganizationByExample,insertUpmsLogSelective,selectUpmsRoleByUpmsUserId,selectUpmsRolePermisstionByUpmsRoleId,selectUpmsSystemByExample,selectUpmsPermissionByUpmsUserIdByCache,selectUpmsUserByUsername,selectUpmsRoleByUpmsUserIdByCache,selectUpmsPermissionByUpmsUserId,selectUpmsUserPermissionByUpmsUserId&mock=true&pid=11444&remote.timestamp=1540453075612&revision=1.0&side=consumer&timeout=10000&timestamp=1540480971509) failed to connect to server /192.168.1.100:20881 client-side timeout 3000ms (elapsed: 3011ms) from netty client 192.168.1.101 using dubbo version 2.5.6
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:127)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:268)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:157)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
